modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -341,3 +342,34 @@ export async function verifySignature(text: string, armoredSignature: string, pu
     return false;
   }
 }
+
+/**
+ * Generate a GPG key pair
+ *
+ * @param: keyCurve the curve to create a key with
+ * @param: username name of the user (optional)
+ * @param: email email of the user (optional)
+ */
+export async function generateGPGKeyPair(
```

Comment:
Can this be implemented and tested into all the places where `.generateKey(` is being used?

I see about 20 calls that could be changed to use this.

Reply:
Not sure this will work for all cases since in tests the curve parameter is often omitted and the default value for it seems to be 'curve25519'. Not sure what the side effects gonna be if I change this.